src/hooks/useCheckTypeGroupOptions.tsx

```diff
@@ -71,6 +71,17 @@ export const CHECK_TYPE_GROUP_OPTIONS: CheckTypeGroupOption[] = [
       // },
     ],
   },
+  {
+    label: `Browser`,
+    description: `Monitor the availability and performance of a website using a real browser.`,
+    value: CheckTypeGroup.Browser,
+    icon: `globe`,
```

Comment:
I added this icon in a rush when I merged `main` in but I think we could do with a better icon. I think we can upload custom ones but I've never looked into it. @VikaCep @majavojnoska what do you think?

![Screenshot of the Browser card on the Choose a check type page. It shows a thick-lined globe icon at the top of the card.](https://github.com/user-attachments/assets/5dced661-e80c-4c37-ae5d-c41e9d698041)

Reply:
We also have this web section icon:

![image](https://github.com/user-attachments/assets/a8ea1cf2-35ce-44b9-bb9b-a6d8b0d03460)

But otherwise we could look into adding a new one. I searched in the `k6` section but I didn't find any icon being used for browser checks there.
Since this is under a feature flag, I believe we can merge and address in a different PR prior to releasing it. Thank you for pointing this out!

Reply:
Created this issue for it https://github.com/grafana/synthetic-monitoring-app/issues/884 so we don't forget.